lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -391,6 +392,46 @@ def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
         end
       end
 
+      # Update the load balancer sticky sessions
+      if sticky_sessions
+        policies = elb.client.describe_load_balancer_policies(load_balancer_name: actual_elb.name)
+        current = policies[:policy_descriptions].detect { |pd| pd[:policy_type_name] == 'AppCookieStickinessPolicyType' }
```

Comment:
It may help fight the 400-line-function-omg-ness of this function (not your fault, was very long before you got here :)) if we move 398-404 into a small method that takes a policy and returns the current_cookie_hash.